distributionviewer/core/static/js/app/components/views/chart-menu.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+export class ChartMenu extends React.Component {
+  render() {
+    return (
+      <nav className="filters">
+        <ul>
+          {this.props.items.map(metric => {
+            return (
+              <li key={`menu-${metric.name}`}>
+                <a href={`/metric/${metric.name}/`}>{metric.name}</a>
```

Comment:
It used to, but it [was removed](https://github.com/reactjs/react-router/issues/1514#issuecomment-122011215). We could consider using [react-router-reverse](https://github.com/ngokevin/react-router-reverse).